cmd/helm/root.go

```diff
@@ -134,31 +135,52 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	flags.ParseErrorsWhitelist.UnknownFlags = true
 	flags.Parse(args)
 
+	groups := templates.CommandGroups{
+		{
+			Message: "Release Commands:",
+			Commands: []*cobra.Command{
+				// release commands
+				newInstallCmd(actionConfig, out),
+				newListCmd(actionConfig, out),
+				newGetCmd(actionConfig, out),
+				newStatusCmd(actionConfig, out),
+
+				newUpgradeCmd(actionConfig, out),
+				newHistoryCmd(actionConfig, out),
+				newRollbackCmd(actionConfig, out),
+				newReleaseTestCmd(actionConfig, out),
+
+				newUninstallCmd(actionConfig, out),
+			},
+		},
+		{
+			Message: "Chart Commands (Beginner):",
```

Comment:
Beginner and Intermediate Chart Commands sounds confusing regarding wrt user experience. I think you could try a little more on segregating this commands. Maybe based on repo usage, then chart verification and packaging maybe.

Reply:
sorry, my bad.  just group release and charts is ok?